src/metric.jl

```diff
@@ -129,13 +129,15 @@ function _rand(
     return r
 end
 
-# TODO The rand interface should be updated by rand from momentum distribution + optional affine transformation by metric
-Base.rand(rng::AbstractRNG, metric::AbstractMetric, kinetic::AbstractKinetic) =
+# TODO The rand interface should be updated as "rand from momentum distribution + optional affine transformation by metric"
+# ignore θ by default unless defined by the specific kinetic (i.e. not position-dependent)
+Base.rand(rng::AbstractRNG, metric::AbstractMetric, kinetic::AbstractKinetic, θ) =
```

Comment:
In this and the below methods, would it make sense to apply similar type bounds to `θ` as above in e.g. `∂H∂θ`?